analyzer/consensus/consensus.go

```diff
@@ -318,11 +339,18 @@ func (m *processor) queueBlockInserts(batch *storage.QueryBatch, data *consensus
 
 func (m *processor) queueEpochInserts(batch *storage.QueryBatch, data *consensusBlockData) error {
 	batch.Queue(
-		queries.ConsensusEpochUpsert,
+		queries.ConsensusFastSyncEpochHeightInsert,
 		data.Epoch,
 		data.BlockHeader.Height,
 	)
```

Comment:
should this only run in fast-sync mode?

Reply:
Yes!! Thank you. I guess technically it wasn't a terrible bug because things would continue to work (as e2e proved also), we'd just create a little extra busy work for the fast-sync finalization stage ... But that's mostly luck :). Fixed!